test/e2e-kind.sh

```diff
@@ -53,7 +58,7 @@ create_kind_cluster() {
 
     # if we are running inside of a kubernetes cluster with /kubepods cgroup
     # being used for pods --> add kubeadmConfigPatches
-    if cat /proc/1/cgroup 2>/dev/null | tail -1 | grep -q /kubepods; then
+    if tail -1 /proc/1/cgroup 2>/dev/null | grep -q /kubepods; then
```

Comment:
yay, one [less useless use of cat](https://en.wikipedia.org/wiki/Cat_(Unix)#Useless_use_of_cat)